targets/x86_64-unknown-none.json

```diff
@@ -0,0 +1,29 @@
+{
+    "llvm-target": "x86_64-unknown-none",
+    "target-endian": "little",
+    "target-pointer-width": "64",
+    "target-c-int-width": "32",
+    "data-layout": "e-m:e-i64:64-f80:128-n8:16:32:64-S128",
+    "arch": "x86_64",
+    "os": "none",
+    "env": "",
+    "vendor": "unknown",
+    "linker-flavor": "gcc",
+    "target-family": "redox",
+    "pre-link-args": {
+        "gcc": ["-m64", "-nostdlib", "-static"]
+    },
+    "features": "-mmx,-sse,-sse2,-sse3,-ssse3,-sse4.1,-sse4.2,-3dnow,-3dnowa,-avx,-avx2,+soft-float",
+    "dynamic-linking": false,
+    "executables": false,
```

Comment:
Unless I change this to `true`, I get the error "cannot produce bin for `oreboot v0.1.0` as the target `x86_64-unknown-none` does not support these crate types".